src/App.js

```diff
@@ -1,5 +1,74 @@
+import { MissionUtils } from "@woowacourse/mission-utils";
+
 class App {
-  async play() {}
+  async play() {
+    let gameOver = false;
+    let correctAnswer;
+
+    MissionUtils.Console.print("숫자 야구 게임을 시작합니다.");
+
+    const answer = [];
+    while (answer.length < 3) {
+      const number = MissionUtils.Random.pickNumberInRange(1, 9);
+      if (!answer.includes(number)) {
+        answer.push(number);
+      }
+    }
+    correctAnswer = answer.join('');
+
+    while (!gameOver) {
+      try {
+        let input = await MissionUtils.Console.readLineAsync("숫자를 입력해주세요: ");
+        input = input.trim();
+
+        if (
+          input.length !== 3 ||
+          input.includes("0") ||
+          input[0] === input[1] ||
+          input[1] === input[2] ||
+          input[0] === input[2]
+        ) {
```

Comment:
인자가 길어지면 함수로 분리하는 게 좋아요!
그리고 각 검사문마다 무슨 일을 하는지 알 수 있게 함수로 한번 더 묶어주면 좋을 것 같습니다.
validateLength, validateZero, validateDupclication 처럼요!

```suggestion
        if (!validateInput(input)) {
```

Reply:
1. 인자가 길어지면 함수로 분리하기
2. 검사문이 무슨 일을 하는지 알 수 있도록 함수로 묶어주기! (네이밍 잘 하도록 하기!)